spec/controllers/groups_controller_spec.rb

```diff
@@ -7,7 +7,7 @@
     it 'create a group when supplied valid params' do
       post :create, group: {
         group_name: 'Group A' }
-      expect(response).to be_success
+      expect(response).to redirect_to action: :index
```

Comment:
hmm, can we set a more specific test here? Seems this would pass before and after your changes.